test/runtests.jl

```diff
@@ -53,12 +53,25 @@ include("test_utils/operations_problem_templates.jl")
 const LOG_FILE = "power-simulations-test.log"
 
 const DISABLED_TEST_FILES = [
-    "test_simulation_sequence.jl",
+    "test_basic_model_structs.jl",
```

Comment:
not all theses tests should be disabled. Only `simulation` and `services` tests should be disabled right now.